tools/src/main/python/dlpx/virtualization/_internal/settings.cfg

```
@@ -15,7 +15,7 @@
 #
 [General]
 engine_api_version = 1.10.5
-package_version = 1.0.0
+package_version = 1.0.2
```

This should be 1.0.1.

```suggestion
package_version = 1.0.1
```